sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -160,9 +118,6 @@ export enum SpanStatusCode {
     UNSET = 0
 }
 
-// @public
-export type TimeInput = HrTime | number | Date;
```

Comment:
Let's just call this `Date` - we don't need to support everything and can assume the caller can make any conversions neeeded. We can always expand later.